psp/exp_configs/uk_pv.py

```diff
@@ -168,13 +168,17 @@ def get_data_source_kwargs(self):
                 coord_system=27700,
                 time_dim_name="init_time",
                 value_name="UKV",
+                y_is_ascending=False,
             )  # , cache_dir=".nwp_cache"),
             # nwp_data_source=None,
         )
 
+    def get_model_config(self) -> PvSiteModelConfig:
+        return PvSiteModelConfig(horizons=Horizons(duration=15, num_horizons=48 * 4))
```

Comment:
does duration not get pulled from the config? when:
(island config, line 54)

        return PvSiteModelConfig(
            horizons=Horizons(
                duration=60,
                num_horizons=48,